### ISSUE TITLE
only use normalization for validation set?

### ISSUE THREAD

**Issue by valencebond** [JunlinHan/CropMix#1]
According to the code of imagenet training, normalization is only used for validation set. In CropMix aug, there is no normalization

**Comment by JunlinHan** [comment]
Hello,
Thanks for your attention in our work.

Please see line 308, the normalization of training data is there.

**Comment by valencebond** [comment]
> Hello, Thanks for your attention in our work.
> 
> Please see line 308, the normalization of training data is there.

Thanks for your quickly reply.
